.github/scripts/Wipe-AlzTenant.ps1

```diff
@@ -22,44 +22,44 @@ $StopWatch.Start()
 $subToMove = Get-AzSubscription -SubscriptionName $subscriptionName
 
 if ($subToMove.State -ne "Disabled") {
-    Write-Information "Moving Subscription: '$($subscriptionName)' under Tenant Root Management Group: '$tenantRootGroupID'"
+    Write-Output "Moving Subscription: '$($subscriptionName)' under Tenant Root Management Group: '$tenantRootGroupID'"
     New-AzManagementGroupSubscription -GroupId $tenantRootGroupID -SubscriptionId $subToMove.Id
 }
 
 
 # For each Subscription in the Intermediate Root Management Group's hierarchy tree, remove all Resources, Resource Groups and Deployments
-Write-Information "Removing all Azure Resources, Resource Groups and Deployments from subscription $($subscriptionName)"
-Write-Information "Set context to SubscriptionId: '$($subToMove.Id)'"
+Write-Output "Removing all Azure Resources, Resource Groups and Deployments from subscription $($subscriptionName)"
+Write-Output "Set context to SubscriptionId: '$($subToMove.Id)'"
 Set-AzContext -Subscription $subToMove.Id #| Out-Null
 
 # Get all Resource Groups in Subscription
 $resources = Get-AzResourceGroup
 
 $resources | ForEach-Object -Parallel {
-    Write-Information "Deleting  $_.ResourceGroupName ..."
+    Write-Output "Deleting  $($_.ResourceGroupName) ..."
     Remove-AzResourceGroup -Name $_.ResourceGroupName -Force | Out-Null
 }
 
 # Get Deployments for Subscription
 $subDeployments = Get-AzSubscriptionDeployment
 
-Write-Information "Removing All Subscription Deployments for: $($subscriptionName)"
+Write-Output "Removing All Subscription Deployments for: $($subscriptionName)"
 
 # For each Subscription level deployment, remove it
 $subDeployments | ForEach-Object -Parallel {
-    Write-Information "Removing $($_.DeploymentName) ..."
+    Write-Output "Removing $($_.DeploymentName) ..."
     Remove-AzSubscriptionDeployment -Id $_.Id
 }
 
 
 # Get all AAD Tenant level deployments
-$tenantDeployments = Get-AzTenantDeployment
+$tenantDeployments = Get-AzTenantDeployment | Where-Object {$PSItem.DeploymentName -like "$intermediateRootGroupID*"}
```

Comment:
is this going to break the clean-ups in the public cloud as they dont have these deployment names yet?

Reply:
It is actually, good catch. I'll rename to mc-Wipe-Alz-Tenant.ps1 and we can merge once we get the naming sorted.